generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,10 @@
+# ===================================================================
+# Spring Cloud Config bootstrap configuration
+# ===================================================================
+
+spring:
+    application:
+        name: <%= baseName %>
+    cloud:
+        config:
+            uri: http://localhost:8761/config
```

Comment:
Yes, at the time of deployment, we need to set the active profile at
manifest.yml / deployemnt config files.

On Wed, Feb 17, 2016 at 3:20 AM, Julien Dubois <notifications@github.com>
wrote:

> In generators/server/templates/src/main/resources/config/_bootstrap.yml
> <https://github.com/jhipster/generator-jhipster/pull/2897#discussion_r53084891>
> :
>
> > @@ -0,0 +1,10 @@
> > +# ===================================================================
> > +# Spring Cloud Config bootstrap configuration
> > +# ===================================================================
> > +
> > +spring:
> > +    application:
> > +        name: <%= baseName %>
> > +    cloud:
> > +        config:
> > +            uri: http://localhost:8761/config
>
> I'm not familiar with bootstrap.yml, could we have one for dev and one for
> prod? As "localhost" would only work for dev
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/2897/files#r53084891>
> .
>



-- 
Thanks 'n Regards
@_sunkara